pkgs/by-name/al/alsa-plugins/package.nix

```
@@ -1,4 +1,14 @@
-{ stdenv, fetchurl, lib, pkg-config, alsa-lib, libogg, libpulseaudio ? null, libjack2 ? null }:
+{ stdenv
+, fetchurl
+, lib
+, pkg-config
+, alsa-lib
+, libogg
+, libsamplerate
+, ffmpeg
+, libpulseaudio ? null
+, libjack2 ? null
```

```suggestion
, libpulseaudio
, libjack2
```
overwriting dependencies with null is highly discouraged